BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -96,15 +96,48 @@ public struct AccessibilityElement: Element {
 
 extension Element {
 
-    /// Wraps the element to provide the passed accessibility
-    /// options to the accessibility system.
+
+    /// Wraps the receiver in an accessibility element with the provided values.
+    ///
+    /// - Important: ⚠️ This overrides the accessibility of the contained element and all of its children ⚠️
+    ///
+    /// - SeeAlso: ``accessibilityElement``
+    @available(
+        *,
+        deprecated,
+        renamed: "accessibilityElement(label:value:traits:hint:identifier:accessibilityFrameSize:)"
+    )
     public func accessibility(
         label: String? = nil,
         value: String? = nil,
         hint: String? = nil,
         identifier: String? = nil,
         traits: Set<AccessibilityElement.Trait> = [],
         accessibilityFrameSize: CGSize? = nil
+    ) -> AccessibilityElement {
+        accessibilityElement(
+            label: label,
+            value: value,
+            traits: traits,
+            hint: hint,
+            identifier: identifier,
+            accessibilityFrameSize: accessibilityFrameSize
+        )
+    }
+
+    /// Wraps the receiver in an accessibility element with the provided values.
+    ///
+    /// Providing a `nil` value for any of these parameters will result in no resolved value for that accessibility
+    /// parameter—it does not inherit parameters from the wrapped element's accessibility configuration.
+    ///
+    /// - Important: ⚠️ This overrides the accessibility of the contained element and all of its children ⚠️
+    public func accessibilityElement(
```

Comment:
`setAccessibility`?

Reply:
`settingAccessibility`?

Reply:
Also kind of a fan of just being very explicit, eg `overrideAccessibility`

Reply:
I think `accessibilityElement` is pretty accurate, since this creates a new accessibility element node. "Set" and "override" both have the same problem in that they kind of imply you're putting pushing updated values onto an existing node.

Reply:
IMO reading just `accessibilityElement` doesn't imply that for me

Reply:
I think `content.accessibility(label: "New Label")` is easy to confuse as just setting accessibility on the content, whereas `content.accessibilityElement(label: "New Label", value: nil, traits: [.staticText])` makes it much more clear that it's wrapping for me.

I'm not sure if it's clear _enough_, and probably not _more_ clear than using the word `override`, but I do think it's just as clear as `setAccessibility` and `settingAccessibility`, personally.

I'll cast a vote for what I have here, but happy to change if anyone feels really strongly (or the majority sways towards another name).

Reply:
Casting a small vote for `overrideAccessibility` or `accessibilityOverride` or similar. The note about `element` making it clear that it's a "wrapping" thing I think is clear for those of us who speak natively in Blueprint element trees, but is less immediately obvious to casual users.

Reply:
IMO it's the required arguments that make any difference; and I think `content.accessibility(label: "New Label", value: nil, traits: [.staticText])` accomplishes the goal without a name change.

If we really want a name change, I prefer `overrideAccessibility` but I don't think any name really solves the issue, it still feels like you're only overriding the properties you're passing in. 

So.. my vote is none of the above? Just make the arguments required (with a deprecated one without that?)

Reply:
Hmm after discussing I think I prefer `accessibilityElement` and if I'm the tie breaker that's my vote. But my preference would be keep it `accessibility`, add permutations of all the default args & deprecate those.

Reply:
Will go with `accessibilityElement(...)` for now, then! I'd really like to get this into a release. We can update the name later—if anyone feels strongly please submit another PR on top of this one so we can discuss further.

Reply:
What about `settingAccessibilityTo`?

Reply:
(If we make all params required; this makes it pretty clear IMO)

Reply:
Gonna go with majority rule (3/5) for this PR and stick with `accessibilityElement(...)`.